runtime/syntax/c.yaml

```diff
@@ -42,7 +42,8 @@ rules:
         end: "'"
         skip: "\\\\."
         rules:
-            - error: "..+"
+              # TODO: Revert back to - error: "..+" once #3127 is merged
+            - error: "[[:graph:]]{2,}'"
```

Comment:
Why need it in `c.yaml`?

Reply:
It doesn't work with something like
```c
const char charArray[] = {'(', ')', ';', '{', '}', '/', '#', ',', '=', '<', '>'};
```
right now